config-connector/tests/ccs-test/cmd/root.go

```diff
@@ -246,10 +246,17 @@ func verifyReadyCondition(solutionPath string) error {
 			continue
 		}
 
+		resourceFilePath := filepath.Join(solutionPath, fileName)
 		output, err := exec.Command("kubectl", "wait", "--for=condition=ready",
-			"-f", filepath.Join(solutionPath, fileName), "--timeout=60s").CombinedOutput()
+			"-f", resourceFilePath, "--timeout=60s").CombinedOutput()
 		if err != nil {
-			return fmt.Errorf("resource in file %q is not ready in 60 seconds: %v\nstdout: %s", fileName, err, string(output))
+			errToReturn := fmt.Errorf("resource in file %q is not ready in 60 seconds: %v\nstdout: %s", fileName, err, string(output))
```

Comment:
should it be configurable? Some resources (SQL etc) will likely take longer than 60s.

Reply:
Yep, I'll make it configurable when I work on the SQL test cases.